pkgs/dimo414/bkt/pkg.yaml

```diff
@@ -1,4 +1,4 @@
 packages:
   - name: dimo414/bkt@0.8.0
   - name: dimo414/bkt
-    version: 0.5.4
+    version: 0.8.0
```

Comment:
You shouldn't update this version.
This version should be 0.5.4 to test the version override.

https://github.com/aquaproj/aqua-registry/blob/7bf78cf13c6dc2a22cd4df07ff19a66c6ae0ee78/pkgs/dimo414/bkt/registry.yaml#L8-L21

And the version 0.8.0 is a duplicate.

https://github.com/aquaproj/aqua-registry/blob/7bf78cf13c6dc2a22cd4df07ff19a66c6ae0ee78/pkgs/dimo414/bkt/pkg.yaml#L2-L4

Reply:
Reverted. bb82a0862f48301d6d9dcdd97d9c91d4887c94b4